src/components/definition-list/definition-list.style.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/prefer-default-export */
+import styled, { css } from 'styled-components';
+import { baseTheme } from '../../style/themes';
+import StyledButton from '../button/button.style';
+import LinkStyle from '../link/link.style';
+
+const theme = baseTheme;
+
+export const StyledDl = styled.dl`
+    display: inline-flex;
+    height: auto;
+    width: 100%;
+    background-color: transparent;
+    overflow: hidden;
+`;
+
+export const StyledDtDiv = styled.div`
+${({ w, dtTextAlign }) => {
```

Comment:
I think it would be better to destructure the theme rather than set to the `baseTheme`

```
${({ w, dtTextAlign, theme }) => {
```

Reply:
That's a fair shout. I tried to avoid destructing it everywhere but it looks like the best way of doing it. Cheers.

Reply:
✅